src/Hosting/Server.IntegrationTesting/src/Common/Tfm.cs

```diff
@@ -13,7 +13,7 @@ public static class Tfm
         public const string NetCoreApp22 = "netcoreapp2.2";
         public const string NetCoreApp30 = "netcoreapp3.0";
         public const string NetCoreApp31 = "netcoreapp3.1";
-        public const string NetCoreApp50 = "netcoreapp5.0";
+        public const string Net50 = "net5.0";
```

Comment:
Could have left the `const` name alone and saved some C# changes. But, this will likely be less confusing even 3 months from now.